scripts/entrypoint-web.sh

```
@@ -1,105 +1,2 @@
-# Wait for Gunicorn to exit (disable errexit so we always reach shutdown)
-set +e
-wait "$GUNICORN_PID"
-GUNICORN_EXIT_CODE=$?
-set -e
-
-echo ""
-echo "⚠ Gunicorn exited with code $GUNICORN_EXIT_CODE"
-
-# Stop Nginx when Gunicorn exits (pass through Gunicorn exit code for orchestration)
-shutdown "$GUNICORN_EXIT_CODE"
+# Entrypoint for web container (nginx + gunicorn via supervisord)
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)